commit 54728e1dbad534190b571211cf932102fabdbf3c
Author: Scott Moser <smoser@brickies.net>
Date:   Thu Mar 7 19:33:05 2013 -0500

    changes to support dokucrypt plugin
    
    * Resolving problems with empty pages and toolbar header buttons
      lib/scripts/edit.js
    
      The header buttons in the editor toolbar are not working.
      Reported by Termiter.

diff --git a/lib/scripts/edit.js b/lib/scripts/edit.js
index 5a5e829..c4892fa 100644
--- a/lib/scripts/edit.js
+++ b/lib/scripts/edit.js
@@ -146,7 +146,7 @@ function currentHeadlineLevel(textboxId){
     var field = jQuery('#' + textboxId)[0],
         s = false,
         opts = [field.value.substr(0,getSelection(field).start)];
-    if (field.form.prefix) {
+    if (field.form && field.form.prefix) { // DOKUCRYPT
         // we need to look in prefix context
         opts.push(field.form.prefix.value);
     }
